docs/user-guide/logging/sdl.md

```diff
@@ -79,7 +82,7 @@ root       479  0.0  0.0   4348   812 ?        S    00:05   0:00 sleep 1
 root       480  0.0  0.0  15572  2212 ?        R    00:05   0:00 ps aux
 ```
 
-What happens if for any reason the image in this pod is killed off and then restarted by Kubernetes? Will we still see the log lines from the previous invocation of the container followed by the log lines for the started container? Or will we lose the log lines from the original container's execution and only see the log lines for the new container? Let’s find out. First let's delete the currently running counter.
+If for any reason the image in this pod is killed off and then restarted by Kubernetes or pod was evicted from the node, logs for the container are lost. Let's check this. First let's delete the currently running counter.
```

Comment:
"If, for any reason, the image in this pod is killed off and then restarted by Kubernetes, or the pod was evicted from the node, logs for the container are lost.

Try deleting the currently running counter container:"